GUI/PyQt/setup.py

```diff
@@ -25,9 +25,9 @@ def readme():
           tests_require=['pytest'],
```

Comment:
Can you please create a second setup file with the current package versions that work for you.
I know for example that numpy 1.16 and sklearn have some unresolved conflicts -> GUI will not open with newest packages